File: python test/memory_test.py
import sys

def memory_test():
    #size test in bytes
    number = 52
    float_number = 52.0
    hello_string = "Hello World"
    print(sys.getsizeof(number))
    print(sys.getsizeof(float_number))
    print(sys.getsizeof(hello_string))

if __name__ == '__main__':
    memory_test()

File: c++ test/memory_test.cpp
#include <iostream>
#include <string>


int main(){
    int number = 52;
    float floatNumber = 52.0f;
    std::string helloString = "Hello World";

    std::cout << sizeof(number) << " " << sizeof(floatNumber) << " " << sizeof(helloString);
}